projects_test.go

```diff
@@ -0,0 +1,32 @@
+package gitlab
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
Not sure if I like using `testify` over just using the standard `Go` approach, but at least there are some test now :wink: